node/src/Pos/Binary/Communication.hs

```diff
@@ -63,6 +67,12 @@ instance (HasConfiguration, SscHelpersClass ssc) => Bi (MsgBlock ssc) where
             1 -> MsgNoBlock <$> decode
             t -> fail $ "MsgBlock wrong tag: " <> show t
 
+-- | Get an encoded MsgBlock from an encoded block.
```

Comment:
The comment is incorrect, you're just constructing the prefix here. You could transform it into function `AsBinary Block -> ByteString` or change the comment.

Reply:
Right, stale comment.